index.html

```diff
@@ -71,25 +71,28 @@ <h3 class="job__title">{{ job.name }}</h3>
         </div>
         {% include scroller.html href="#contact-us" %}
     </div>
+    {% include picture.html cssClass="hunting__image-bottom" alt="Hunting" small="/images/sections/foliage-S-1x.jpg, /images/sections/foliage-S-2x.jpg 2x, /images/sections/foliage-S-3x.jpg 3x" %}
 </div>
 
 <div id="contact-us" class="contacts section">
     <div class="container h-card">
```

Comment:
h-card luokan pitäisi sisältää aina yksi kontakti, nyt h-cardin sisällä on kaksi kontaktia. Kannattaa laittaa h-card luokkamääritys for loopin ensimmäiselle elementille.